libraries/chain/controller.cpp

```diff
@@ -960,7 +960,7 @@ struct controller_impl {
    named_thread_pool<chain>        thread_pool;
    deep_mind_handler*              deep_mind_logger = nullptr;
    bool                            okay_to_print_integrity_hash_on_stop = false;
-   bool                            testing_allow_voting = false; // used in unit tests to create long forks or simulate not getting votes
+   bool                            testing_allow_voting = true; // used in unit tests to create long forks or simulate not getting votes
```

Comment:
Is this correct to set default to true?

Reply:
Nope, quick response to a failed test. Addressing now. Thanks @greg7mdp & @linh2931

Reply:
Later we should have a test to verify old blocks are not voted.